File: exercises/rb101_119/easy1/ex3.rb
def stringy(num, start = 1)
  string = ''

  num.times do |int|
    int += 1 if start == 0
    string << (int.even? ? '1' : '0')
  end
  string
end

puts stringy(6) #== '101010'
puts stringy(9) #== '101010101'
puts stringy(4) #== '1010'
puts stringy(7) #== '1010101'

puts stringy(6,0) #== '101010'
puts stringy(9,0) #== '101010101'
puts stringy(4,0) #== '1010'
puts stringy(7,0) #== '1010101'

File: exercises/js210_small_problems/easy_2/ex3.js
function stringy(num) {
  let string = ''
  let one = true;

  for (; num > 0; num -= 1) {
    string += (one ? 1 : 0);
    one = !one
  }

  return console.log(string);
}

stringy(6);    // "101010"
stringy(9);    // "101010101"
stringy(4);    // "1010"
stringy(7);    // "1010101"
